packages/terra-application-utility/src/header/ApplicationHeaderUtility.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import IconChevronDown from 'terra-icon/lib/icon/IconChevronDown';
+import UtilityMenu from './_HeaderUtilityMenu';
+import Utils from '../_Utils';
+import styles from './ApplicationHeaderUtility.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The array containing the menu items to be rendered within the menu.
+   */
+  menuItems: PropTypes.arrayOf((Utils.itemShape)).isRequired,
+  /**
+   * The function to trigger when a menu item is selected.
+   */
+  onChange: PropTypes.func.isRequired,
+    /**
+   * The function that discloses the menu.
+   */
+  onDisclose: PropTypes.func.isRequired,
+  /**
+   * The name to be displayed next to the user photo.
+   */
+  userName: PropTypes.string,
+  /**
+   * The image associated with the user.
+   */
+  userPhoto: PropTypes.element,
+};
+
+const defaultProps = {
+  menuItems: [],
+  userName: '',
+  userPhoto: undefined,
+};
+
+class ApplicationHeaderUtility extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleOnClick = this.handleOnClick.bind(this);
+    this.createContent = this.createContent.bind(this);
+  }
+
+  handleOnClick() {
+    if (this.props.onDisclose) {
+      const content = this.createContent();
+      this.props.onDisclose(content);
+    }
+  }
+
+  createContent() {
+    return (
+      <UtilityMenu
+        initialSelectedKey={Utils.KEYS.MENU}
```

Comment:
Update this to take a "selectedKey" prop. Utilities should be a controller component.